### ISSUE TITLE
Some question about this nice job

### ISSUE THREAD

**Issue by wangwxr** [developer0hye/Simple-PyTorch-Deformable-Convolution-v2#27]
modulator = 2. * torch.sigmoid(self.modulator_conv(x))  
what's the reason of 2.*torch.sigmoid

**Comment by developer0hye** [comment]
Hi, @wangwxr 

I just followed [original implementation](https://github.com/apache/mxnet/blob/5722f8b38af58c5a296e46ca695bfaf7cff85040/python/mxnet/gluon/nn/conv_layers.py#L1613). I am also curious about it. I thought it is multiplied so that the mean of the `modulator` becomes 1 during the initial training stage.
